regress/pesterTests/SSH.Tests.ps1

```diff
@@ -330,9 +315,10 @@ Describe "E2E scenarios for ssh client" -Tags "CI" {
             $goodConfigFile = Join-Path $testDir "$tC.$tI.Очень_хорошо_ssh_config"
             "#this is a Unicode comment because it contains русский язык" | Set-Content $goodConfigFile -Encoding UTF8
             "Host myhost" | Add-Content $goodConfigFile
-            "    HostName $server" | Add-Content $goodConfigFile
+            "    HostName localhost" | Add-Content $goodConfigFile
```

Comment:
Still call it $server and set it to localhost in BeforeAll. That way we can adjust $server later for Multi Machine testing

Reply:
sure